commands/permissions.go

```diff
@@ -44,11 +46,37 @@ var ShowRoleCmd = &cobra.Command{
 	RunE:    withClient(showRoleCmdF),
 }
 
+var AssignUsersCmd = &cobra.Command{
+	Use:   "assign [role_name] [username...]",
```

Comment:
I think that, as you commented in the PR thread, this belongs to its own subcommand `permission role assign`. Right now it could sound like assigning permissions to users, and as these commands operate over the role entity, I think it makes total sense to create a `role` subcommand

Reply:
Then we should move the `permissions show [role_name]` over to `permissions role show [role_name]` too.

Reply:
Agreed and updated. Thanks.